### DISCUSSION TITLE
KerykeionPoint issues

### DISCUSSION THREAD

**Discussion by mmkeyes140** [g-battaglia/kerykeion#73] [Q&A]
Hello!  I am using Kerykeion to return a list of details based on an individual's natal chart.  I keep running into the following error:
           TypeError: 'KerykeionPoint' object is not callable

This error is popping up when I attempt to call on the data generated in Kerykeion.  This error is appearing on the following attempts:

- Building CSV with natal chart data
- Building JSON with natal chart data
- Using _user.planet_ to look up details in Kerykeion generated data.

Does anyone have suggestions on how to get past this issue?

**Comment by g-battaglia** [comment]
Hi! Could you please provide a code example of the part where you are encountering the 'TypeError: 'KerykeionPoint' object is not callable' error? This will help me better understand the context and provide you with a more specific solution. Thanks!

**Comment by mmkeyes140** [comment]
@g-battaglia thank you for answering my question in a separate email string.   For others stuggling running against the same KerykeionPoint error, I was calling the object using the incorrect method.   

To correct, I needed to change my how I was requesting the from the planetary calculations:.
### The incorrect method: 
```
planetDetails = [
    client.sun, client.moon, client.mercury,
    client.venus, client.mars, client.jupiter,
    client.saturn, client.uranus, client.neptune,
    client.pluto, client.mean_node
    ]
```
```
    for planet_method in planetDetails:
        if planet_method['name'] == planet:
            planet_data = **planet_method('sign')** ## error here
            break
```
### Corrected method

```
for planet_object in planetDetails:
        if planet_object['name'] == planet:
            planet_data = planet_object.sign  ## correction here
            break
```
